test/text-fragment-utils-test.js

```diff
@@ -12,6 +12,27 @@ const marksArrayToString = (marks) => {
   return text.join('').replace(/[\t\n\r ]+/g, ' ').trim();
 };
 
+// A helper function to extract the color and background color from
+// css class the chrome-target-text.
```

Comment:
nit: update to reflect the new name (and remove "chrome" mentions from variable names below)